views/js/controller/creator/index.js

```diff
@@ -95,14 +95,14 @@ define([
             //load plugins dynamically
             if (config) {
                 if(config.plugins){
-                    _.forEach(config.plugins, function (plugin) {
```

Comment:
Any reason to use the foreach func from lodash instead of the one from array prototype?

Reply:
yes plugins can be an array or an object